lib/evmone/analysis.cpp

```diff
@@ -77,19 +77,12 @@ code_analysis analyze(evmc_revision rev, const uint8_t* code, size_t code_size)
 
         auto& instr = analysis.instrs.back();
 
-        bool is_terminator = false;  // A flag whenever this is a block terminating instruction.
-        switch (opcode)
+        switch (opcode_info.kind)
         {
-        case OP_JUMP:
-        case OP_JUMPI:
-        case OP_STOP:
-        case OP_RETURN:
-        case OP_REVERT:
-        case OP_SELFDESTRUCT:
-            is_terminator = true;
+        default:
```

Comment:
I'd say explicit cases here would more clearly show the intention
```suggestion
        op_kind::regular:
        op_kind::terminator:
```